### ISSUE TITLE
4.17.5 broken: Error: Dojo loader version does not match the version of Dojo.

### ISSUE THREAD

**Issue by Biboba** [Esri/arcgis-webpack-plugin#112]
My application built with @arcgis/webpack-plugin 4.17.5 can no longer be built.

## Description

When running npm start, the following error is logged and the build fails:
```
Error: Dojo loader version does not match the version of Dojo.
Loader version = 1.16.3.
Dojo version = 1.16.4.
You may need to rebuild the Dojo loader.
Refer to https://github.com/OpenNTF/dojo-webpack-plugin/blob/master/README.md#building-the-dojo-loader

```

## Expected Behavior

Application built successfully.

## Actual Behavior

Build fails

## Possible Fix

https://github.com/Esri/arcgis-webpack-plugin/issues/100

Thanks !

## Your Environment

* Version used: 4.17.5
* Browser Name and version:
* Operating System and version (desktop or mobile):
* Link to your project if available:

**Comment by odoe** [comment]
Thanks, I backported this fix to 4.17.6. Give that a try.

**Comment by Biboba** [comment]
It works ! Thanks for the quick fix !
